net/ip.go

```diff
@@ -0,0 +1,141 @@
+package net
+
+import (
+	"fmt"
+	"net"
+)
+
+// IP represents an IPv4 or IPv6 address
+type IP struct {
+	higher    uint64
+	lower     uint64
+	ipVersion uint8
+}
+
+// IPv4 returns a new `IP` representing an IPv4 address
+func IPv4(val uint32) IP {
+	return IP{
+		lower:     uint64(val),
+		ipVersion: 4,
+	}
+}
+
+// IPv4FromOctets returns an IPv4 address for the given 4 octets
+func IPv4FromOctets(o1, o2, o3, o4 uint8) IP {
+	return IPv4(uint32(o1)<<24 + uint32(o2)<<16 + uint32(o3)<<8 + uint32(o4))
+}
+
+// IPv6 returns a new `IP` representing an IPv6 address
+func IPv6(higher, lower uint64) IP {
```

Comment:
IPv6FromString() wäre noch schick *find*

Reply:
will be added in a further PR, this one was big enough ;).
In the next step the NLRI will be changed, I can introduce a FromString then.